### PR TITLE
Ecosystem landing

### PR DESCRIPTION
**Pull request by gtca**
Make ecosystem more prominent on the landing page

### CODE REVIEW

**Comment by grst** [comment]
Would be neat to have some picture for the ecosystem on the landing page as well. 

I could think of
 * a bunch of logos
 * the figure from the paper
 * the figure from @ivirshup's SCG poster

**Comment by grst** [comment]
I also think the link out is rather small. It doesn't look like I want to click it.

File: content/_index.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ description = "Foundational tools for single-cell omics data analysis"
 text = """\
 **scverse** is a consortium of foundational tools (mostly in Python) for omics data in life sciences. It has been founded to ensure the long-term maintenance of these core tools."""
 
+# Ecosystem
+
+[ecosystem]
+
+text = """\
+Many packages rely on the functionality of the **scverse** core packages. They implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, etc. We are also building [a listing of such packages](/packages#ecosystem) that follow development best practices."""
```
Comment (ivirshup) [inline content/_index.md:54]:
```suggestion
A broader ecosystem of packages builds on the **scverse** core packages. [These tools](/packages#ecosystem) implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, and more.
```

**Review by ivirshup — COMMENTED** [review]
I think it's important we have something up, but a very good representation could take more thought.

Unless there's something very concrete we'd wanna do, maybe lets merge the text solution and discuss in an issue or zulip?

**Review by grst — APPROVED** [review]
Agreed, let's go!

File: content/_index.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ description = "Foundational tools for single-cell omics data analysis"
 text = """\
 **scverse** is a consortium of foundational tools (mostly in Python) for omics data in life sciences. It has been founded to ensure the long-term maintenance of these core tools."""
 
+# Ecosystem
+
+[ecosystem]
+
+text = """\
+A broader ecosystem of packages builds on the **scverse** core packages. [These tools](/packages#ecosystem) implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, and more.
```
Comment (grst) [inline content/_index.md:54]:
```suggestion
A broader ecosystem of packages builds on the **scverse** core packages. [These tools](/packages#ecosystem) implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, and more."""
```

**Pull request merged**